include/ws_functions.inc.php

```diff
@@ -158,7 +158,14 @@ function ws_flickr2piwigo_importPhoto($params)
       else
       {
         $logger->info('Creating category: '.$photoset_info['title'], FLICKR2PIWIGO);
-        $cat = create_virtual_category(pwg_db_real_escape_string($photoset_info['title']));
+        $photoset_info = $flickr->photosets()->getInfo($photoset_info['id'], $params['user_id']);
+        $photoset_options=NULL;
+        if (isset($photoset_info['description'])) {
+          $photoset_description = $flickr->photosets()->getInfo($photoset_info['id'], NULL)['description'];
```

Comment:
`getInfo()` is being called twice. This line could be left out and `$photoset_info['description']` used below instead.